models/reporter.go

```diff
@@ -79,6 +79,13 @@ SUM(num_queries) AS num_queries,
 rowNumberInAllBlocks() AS total_rows
 FROM metrics
 WHERE period_start > :period_start_from AND period_start < :period_start_to
+{{ if .Search }}
+	{{ if eq .Group "queryid" }}
+		AND ( lowerUTF8(queryid) LIKE :search OR lowerUTF8(metrics.fingerprint) LIKE :search )
```

Comment:
`queryid` without metrics, but fingerprint with metrics. Why?

Reply:
Because of the name conflict. `fingerprint` is used in SELECt statement.